README.md

```diff
@@ -1,12 +1,18 @@
-# channumerals
+# CLDF dataset derived from Chan's Numerals
```

Comment:
Add a year and quotation marks, a la https://github.com/lexibank/kitchensemitic/blob/master/README.md